jest.config.js

```diff
@@ -1,171 +1,9 @@
-// No need to use ts-jest https://github.com/vercel/next.js/discussions/13528#discussioncomment-22933
-// Client-side Jest config
-// You might need to setup another config file for server tests
-module.exports = {
```

Comment:
@Timi-Duban could you please leave the commented options? They help a lot during development